index.html

```diff
@@ -45,5 +47,247 @@ <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                <h2>Tonic</h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <div class="work4">
+
+            <div>
+                <a>
+                    <img src="img/work4.png" alt="work image">
+                </a>
+            </div>
+            <div class="work1-title">
+                <h2>Multi-Post <br>
+                    Stories<br>
+                </h2>
+            </div>
+            <div class="work1-detail">
+                <ul>
+                    <li><a class="text-decoration">CANOPY</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">Back End Dev</a></li>
+                    <li><img src="img/circle.png" alt="circle image"></li>
+                    <li><a class="text-decoration">2015</a></li>
+                </ul>
+            </div>
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration blue-color">HTML</a></li>
+                    <li><a class="text-decoration blue-color">CSS</a></li>
+                    <li><a class="text-decoration blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+        <section class="about-myself">
+            <div class="about-myself-container">
+                <div class="title">
+                    <h2>About Myself</h2>
+                </div>
```

Comment:
- Kindly, update the about section color to `#172B4D`, to stick the Figma design :+1: as the previous code reviewer suggested you